react/e2e/private/homeScreen/transactionRequest.test.js

```diff
@@ -32,14 +37,49 @@ describe('transaction request', async () => {
     await creditButton.click()
 
     // Request transacton
-    const requestTransactionButton = await page.$(
-      requestTransactionButtonSelector
-    )
-    requestTransactionButton.click()
+    await page.click(requestCreditTransactionBtn)
 
     await page.waitForNavigation({ waitUntil: 'networkidle2', timeout: 5000 })
 
     // redirect to historyScreen
-    await expect(page.url()).toMatch('/requests')
+    await expect(page.url()).toMatch(REQUEST_URL)
   }, 20000)
+
+  it('displays stored transaction', async () => {
+    // Login as Person2
+    const page2 = await browser.newPage();
+    await login(page2,'Person2', 'password')
+    await page2.goto(REQUEST_URL)
+    await page2.waitForSelector(requestItemSelector);
+
+    // Get last transaction ID
+    const lastItemId1 = await page2.$eval(requestItemSelector, e => e.getAttribute('data-request-id'));
+
+    await login(page,'Person1', 'password')
+
+    await page.type(recipientSelector, 'Person2')
+
+    // Select credit type transaction
+    await page.click(debitSelector)
+
+    // Create transaction
+    await addTransaction(page, milk)
+    await addTransaction(page, honey)
+    await addTransaction(page, bread)
+    await getTotal()
+
+    // Request transacton
+    await page.click(requestDebitTransactionBtn)
+    await page.waitForNavigation({ waitUntil: 'networkidle2', timeout: 5000 })
+
+    await page2.reload()
+    await page2.waitForSelector(requestItemSelector);
+    const lastItemId2 = await page2.$eval(requestItemSelector, e => e.getAttribute('data-request-id'));
+
+    await page2.close()
+    await login(page)
+
+    // Assert last transaction ID has changed
+    expect(lastItemId1).not.toEqual(lastItemId2)
```

Comment:
current test expects **any** new request instead of [debit request from step 2](https://drive.google.com/file/d/1LI29q5nxHIGfSzpd9-7fjqasOJiVuyQ9/view?usp=sharing) (expected from issue #98)

please replace items with single, non-standard `const expectedPrice = (Math.random() * Math.floor(100)).toFixed(3)`, then pass `expectedPrice` as argument to matcher